samples/tfm_integration/tfm_secure_partition/dummy_partition/tfm_manifest_list.yaml.in

```diff
@@ -0,0 +1,30 @@
+#-------------------------------------------------------------------------------
+# Copyright (c) 2018-2021, Arm Limited. All rights reserved.
+#
+# SPDX-License-Identifier: BSD-3-Clause
+#
+#-------------------------------------------------------------------------------
+
+{
+  "name": "TF-M secure partition manifests",
+  "type": "manifest_list",
+  "version_major": 0,
+  "version_minor": 1,
+  "manifest_list": [
+    {
+      "name": "Dummy Partition",
+      "short_name": "TFM_DP",
+      "manifest": "${ZEPHYR_BASE}/samples/tfm_integration/tfm_secure_partition/dummy_partition/tfm_dummy_partition.yaml",
+      "source_path": "${ZEPHYR_BASE}/samples/tfm_integration",
+      "tfm_partition_ipc": true,
```

Comment:
Generate this from a Kconfig perhaps?

Reply:
Never mind, I misunderstood. Looks good.

- ``tfm_partition_ipc``: indicate if this partition is compatible with the IPC
  model.